### ISSUE TITLE
Low CIFAR10 accuracy

### ISSUE THREAD

**Issue by ayseserbetci** [htqin/IR-Net#11]
Hello,
Thank you very much for sharing your work.
When I trained ResNet18 for CIFAR10, it gives 86% accuracy on the validation data.
I use the same hyperparameters of  https://github.com/kuangliu/pytorch-cifar as you point out in the paper.
